src/Bot.ts

```diff
@@ -152,4 +155,16 @@ export class Bot extends builder.UniversalBot {
             callback(null, null, 200);
         };
     }
+
+    // set incoming event to any because membersAdded is not a field in builder.IEvent
+    private getMessageReactionsHandler(bot: builder.UniversalBot): (event: any) => void {
```

Comment:
Talk to Josh about this-- we don't want to promote this pattern of having a getXXXHandler that returns a function. It should be:

private handleMessageReaction(event: builder.IMessageUpdate): void {
...
}

this.on("messageReaction", this.handleMessageReaction.bind(this));

Reply:
NIT: arrow function is more suggested than `bind()`

Reply:
Ah yes, Amit, let’s use an arrow function.

Reply:
Ah yes, Amit, let’s use an arrow function.

Reply:
Done.